#goal definition
int8 robotPose
---
#result definition
int8 robotPose
---
#feedback
